### ISSUE TITLE
[BUG] the resume point window is displayed for each resume point

### ISSUE THREAD

**Issue by Titistajn** [nixgates/plugin.video.seren#509]
If playback is interrupted twice, e.g. at 15:06 and 40:02, after selecting the source, a window will open offering a resume from 40:02. After confirmation, a window will open offering a resume from 15:06. If 15:06 is selected, playback will resume from 40:02. 

**Expected behavior**
The resume window opens only once for the last resume point.

**Kodi Version
 - Kodi Version 18.9 (CoreELEC 9.2.5)
 - Version 2.0.16

**Comment by LoganWolverineX** [comment]
I've noticed something similar too.  I've noticed sometimes  i would pause  one episode somewhere in the middle, then come back to resume and watch it.  It resumes just fine.  I then finish the episode, but then some time later either after the skin refreshes or after coming back into kodi, the episode i finished from the resume, still shows up as in progress.  I would just have to reset progress from trakt manager and move on.  Sometimes, it happens more than once to the same episode I already finished.  I think it's similar what you're experiencing.  It seems like the Trakt scrobbling isn't being updated after the first resume?
